test/test_CE6SSM_gmm2.cpp

```diff
@@ -20,30 +20,34 @@
 #define BOOST_TEST_MODULE test_CE6SSM_gmm2
 
 #include <boost/test/unit_test.hpp>
-#include <cstdlib>
+//#include <cstdlib>
 
-#include "test.hpp"
+//#include "test.hpp"
 #include "test_CE6SSM.hpp"
 
-#include "lowe.h"
-#include "wrappers.hpp"
+//#include "lowe.h"
+//#include "wrappers.hpp"
 #include "CE6SSM_a_muon.hpp"
-#include "CE6SSM_mass_eigenstates.hpp"
-#include "CE6SSM_susy_parameters.hpp"
-#include "CE6SSM_semi_analytic_ewsb_solver.hpp"
-#include "CE6SSM_semi_analytic_solutions.hpp"
+//#include "CE6SSM_mass_eigenstates.hpp"
+//#include "CE6SSM_susy_parameters.hpp"
+#include "physical_input.hpp"
+#include "CE6SSM_semi_analytic_spectrum_generator.hpp"
+//#include "CE6SSM_semi_analytic_ewsb_solver.hpp"
+//#include "CE6SSM_semi_analytic_solutions.hpp"
```

Comment:
Just saw this push and the changes look odd.  Did you mean to push it as ready with these commented out lines and a cout statement or you are checking the test just now?  Anyway we can discuss it when i speak to you shortly.

Reply:
Hi Peter, I don't believe I've got the test quite up to scratch yet, so in preparation for our discussion I just upload a commit with "preliminary" work.